File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/createnew/fragments/createnew/view/CreateNewView.java
package ua.meugen.android.pdfcomposer.ui.activities.createnew.fragments.createnew.view;

import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.view.MvpView;


public interface CreateNewView extends MvpView {

    void loadImage();

    void addPage();

    void exportToPdf();

    void displayContent(String imageUri, String text);

    void pageDidntComplete();
}

File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/createnew/fragments/createnew/view/CreateNewView.kt
package ua.meugen.android.pdfcomposer.ui.activities.createnew.fragments.createnew.view

import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.view.MvpView


interface CreateNewView : MvpView {

    fun loadImage()

    fun addPage()

    fun exportToPdf()

    fun textChanged(text: CharSequence)

    fun displayContent(imageUri: String?, text: String?)

    fun pageDidntComplete()
}
